packages/core/core/src/AssetGraphBuilder.js

```diff
@@ -187,6 +160,20 @@ export default class AssetGraphBuilder extends EventEmitter {
       }
     }
   }
+
+  isInvalid() {
+    return !!this.graph.invalidNodes.size;
+  }
+
+  respondToFSEvents(events: Array<Event>) {
+    for (let {type, path} of events) {
+      // TODO: eventually handle all types of events
+      if (type === 'update' && this.graph.hasNode(path)) {
+        this.controller.abort();
```

Comment:
does aborting this signal synchronously cause an error to throw? if it does, it would prevent other files in this event from getting invalidated

Reply:
Looks like that won't be an issue. It's kind of weird the signal can be aborted multiple times, but that's probably fine too. Could move it out of the loop if you want.

Reply:
No, it doesn't throw anything it just sets the `signal.aborted` property to true. I don't think it would hurt anything, but it is kind of weird that it might be called multiple times now that we are receiving multiple events at once. I decided to move it to the beginning of the build method, where I think it makes a little more sense.